README.md

```diff
@@ -0,0 +1,55 @@
+`tf_lambda_s3_backup`
+---------------------
+
+This modules allows configure a ECS task with a given container to be triggered using
+a cron expression.
+
+Currently is designed to trigger backups using the container: https://github.com/mergermarket/docker-mysql-s3-backup,
+but it would be adapted to run any container.
+
+*Note*: since the release of [scheduled ECS tasks from AWS](https://aws.amazon.com/about-aws/whats-new/2017/06/amazon-ecs-now-supports-time-and-event-based-task-scheduling), there is no point to use a lambda for this anymore. But the code related to tasks can be reused.
```

Comment:
Is it worth considering whether to transform this module to use Time and Even based task scheduling?